backend/db.js

```diff
@@ -1,8 +1,8 @@
 //db.js
-
+const mongoose = require('mongoose');
 require('dotenv').config();
 
-const mongoose = require('mongoose');
+
 
 const dbUri = `mongodb://${process.env.DATABASE_USER}:${process.env.DATABASE_PASSWORD}@${process.env.DATABASE_HOST}:${process.env.DATABASE_PORT}/${process.env.DATABASE_NAME}`;
```

Comment:
Is this string still in use or now using the environtent variable "process.env.MONGO_URL"?